lib/client.js

```diff
@@ -285,14 +321,7 @@ extend(Raven.prototype, {
     // todo: better property name than sentryContext, maybe __raven__ or sth?
     wrapDomain.sentryContext = options;
 
-    var self = this;
-    if (typeof onErr !== 'function') {
-      onErr = function (err) {
-        self.captureException(err);
-      };
-    }
-
-    wrapDomain.on('error', onErr);
+    wrapDomain.on('error', this.makeErrorHandler());
```

Comment:
what are the implications of having multiple makeErrorHandler() closures floating around? aren't they there to do some sort of de-duplication?